### PR TITLE
Add exception inheritance on import settings module. Add initial error message.

### PR DESCRIPTION
**Pull request by cthtuf**
Hi! 
I like your project and use it in all my non-django projects. Thank you for it!

From time to time I'm facing with issue when settings file contains errors. For example I'm using python-decouple and when I don't specify default value for env and it wasn't passed properly, then I get ImproperlyConfigured exception, but it's not so informative. I suggest to add a bit details which would make it easier to debug and fix.
It would be great to have this changes in a new version.
Here is screenshots with logs to see the difference:

Before: 
image

---
After:
image

### CODE REVIEW

**Comment by charlsagente** [comment]
Hi @cthtuf, I'm glad you found this project useful, I use it in all my non-django projects too!
Thanks for creating a PR, I will review it, test it and we can continue from there.

File: python_settings/__init__.py [hunk 1]
```diff
@@ -59,10 +59,10 @@ def __init__(self, settings_module):
             self.SETTINGS_MODULE = settings_module
             mod = importlib.import_module(
                 self.SETTINGS_MODULE)
-        except ImportError:
-            raise ImproperlyConfigured("Cannot import SETTINGS_MODULE")
-        except Exception:
-            raise ImproperlyConfigured("Error trying to import your settings module")
+        except ImportError as ie:
+            raise ImproperlyConfigured(f"Cannot import SETTINGS_MODULE: {ie}") from ie
```
Comment (charlsagente) [inline python_settings/__init__.py]: Understood. I'd like to ensure we're considerate of those still using Python 2.7 and versions <= 3.5, as there are still some users relying on them. What approach can we take to ensure backward compatibility?

**Review by charlsagente — CHANGES_REQUESTED** [review]

File: python_settings/__init__.py (hunk 1 above)
Comment (cthtuf) [reply python_settings/__init__.py]: Sure, updated the PR. I removed exception inheritance and replaced f-string with formatted string. Tested it locally with python2.7

**Review by charlsagente — APPROVED** [review]
Looking good, I tested on different Python versions. 
I can add test coverage in the future for the generic Exception case.

**Pull request merged**